modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/input_move_boxes.js

```diff
@@ -140,25 +142,24 @@ AUI.add(
 				_onSelectChange(event) {
 					const instance = this;
 
-					instance._toggleBtnMove(event);
-					instance._toggleBtnSort(event);
+					instance._toggleBtnMove(event.currentTarget);
+					instance._toggleBtnSort(event.currentTarget);
 				},
 
 				_onSelectFocus(event, box) {
 					const instance = this;
 
-					instance._toggleBtnMove(event);
+					instance._toggleBtnMove(event.currentTarget);
```

Comment:
same here

Reply:
please check my comment for the same issue above.